docs/guide/previews.md

```diff
@@ -33,9 +33,15 @@ class ExampleComponentPreview < ViewComponent::Preview
 end
 ```
 
-Which generates <http://localhost:3000/rails/view_components/example_component/with_default_title>,
-<http://localhost:3000/rails/view_components/example_component/with_long_title>,
-and <http://localhost:3000/rails/view_components/example_component/with_content_block>.
+Generate the preview using:
```

Comment:
I think `generate` was the wrong word to use in the existing docs. It should instead say something like `which can then be accessed at:`